### PR TITLE
[3.7.0] docs(scheduler): OnCellRender event

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/scheduler/events.md [hunk 1]
```diff
@@ -939,6 +940,106 @@ Through its event arguments you can get the `Item` to cast it to your model type
 </style>
 ````
 
+## OnCellRender
+
+The `OnCellRender` event fires upon rendering of each slot cell. It allows you set custom CSS class to the cells based on the business logic.
```
Comment (dimodi) [inline components/scheduler/events.md:945]:
```suggestion
The `OnCellRender` event fires upon rendering of each slot cell. It allows you to set a custom CSS class to the cell, based on business logic.
```

File: components/scheduler/events.md [hunk 2]
```diff
@@ -939,6 +940,106 @@ Through its event arguments you can get the `Item` to cast it to your model type
...
+
+The handler receives an argument of type `SchedulerCellRenderEventArgs` which exposes the following fields:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Field | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | The CSS class that will be applied to the cells. The CSS rules that are set for that class will be visibly rendered on the Scheduler cells. |
```
Comment (dimodi) [inline components/scheduler/events.md:953]: The second sentence is self-evident, and at the same time, it guarantees something that is not that certain :)

```suggestion
| `Class` | `string` | The CSS class that will be applied to the cell. |
```

File: components/scheduler/events.md [hunk 3]
```diff
@@ -939,6 +940,106 @@ Through its event arguments you can get the `Item` to cast it to your model type
...
+| `Date` | `DateTime` | The actual date that is associated with the cell. |
```
Comment (dimodi) [inline components/scheduler/events.md:954]: What additional value does "actual" bring?

**Review by dimodi — APPROVED** [review]

File: components/scheduler/events.md [hunk 4]
```diff
@@ -939,6 +940,106 @@ Through its event arguments you can get the `Item` to cast it to your model type
...
+The `OnCellRender` event fires upon rendering of each slot cell. It allows you to set a custom CSS class to the cell, based on business logic.
+
+The handler receives an argument of type `SchedulerCellRenderEventArgs` which exposes the following fields:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Field | Type | Description |
+| --- | --- | --- |
+| `Class` | `string` | The CSS class that will be applied to the cell. |
+| `Date` | `DateTime` | The actual date that is associated with the cell. |
```
Comment (ntacheva) [inline components/scheduler/events.md:954]:
```suggestion
| `Date` | `DateTime` | The date that is associated with the cell. |
```

File: components/scheduler/events.md (hunk 3 above)
Comment (ntacheva) [reply components/scheduler/events.md]: It doesn't, I removed it.

**Pull request merged**